File: 1.arrays/rotate-array/rotateArray.js
function leftRotate(arr, d) {
  const first = arr.slice(0, d);
  const rest = arr.slice(d, arr.length - 1);
  return [].concat(rest, first);
}
const arr1 = [1, 2, 3, 4, 5, 6, 7];
console.log(leftRotate(arr1, 2));

File: 1.arrays/rotate-array/rotate_array.py
# Write a function rotate(ar[], d, n) that rotates arr[] of size n by d elements
def left_rotate(arr, d):
  first, rest = arr[:d], arr[d:-1]
  return rest+first


arr1 = [1,2,3,4,5,6,7]
print(left_rotate(arr1,2))